src/components/Layout/Layout.module.scss

```diff
@@ -0,0 +1,14 @@
+.main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  min-width: 100vw;
+  height: 100%;
+  background-color: #ffe9e0;
```

Comment:
I won't repeat comments on the same issue that was pointed out. But please don't forget about these colors as well :)